pkg/v1/remote/write.go

```diff
@@ -37,6 +39,9 @@ type manifest interface {
 	Digest() (v1.Hash, error)
 }
 
+const maxRetries = 2
```

Comment:
Can you add a comment about what gets retried this many times? Alternatively, just move them into the method where they're used.